src/domains/integration/controller/integration.controller.ts

```diff
@@ -35,16 +34,17 @@ const administratorRepository: AdministratorRepository = new AdministratorReposi
 const integrationRepository: IntegrationRepository = new IntegrationRepositoryImpl(db);
 const service: IntegrationService = new IntegrationServiceImpl(adapter, projectRepository, userRepository, pendingAuthRepository, pendingMemberMailsRepository, organizationRepository, administratorRepository, integrationRepository, emailService);
 
-integrationRouter.post('/linear/projects', validateRequest(ProviderKeyDTO, 'body'), async (req: Request, res: Response) => {
-  const { key, provider } = req.body as unknown as ProviderKeyDTO;
+integrationRouter.post('/:issueProviderName/projects', validateRequest(IssueProviderURLParams, 'params'), validateRequest(ApiKeyDTO, 'body'), async (req: Request<any, any, ApiKeyDTO>, res: Response) => {
```

Comment:
Frontend is sending an object with two different properties in the body of the request and ApiKeyDTO only accepts one. It is triggering a validation exception from the validation class middleware. Please correct our DTO or contact frontend team so they fix what they are sending